src/presence_analyzer/utils.py

```diff
@@ -199,3 +199,65 @@ def star_end_time(data, user_id):
         result[day]['end'] = average_seconds(result[day], 'end')
 
     return result
+
+
+def bussines_days(year_month):
+    """
+    Calculate soconds of worked days in month.
+    """
+    year, month = year_month
+    eight_hour = 28800
+    date = datetime(
+        year,
+        month,
+        calendar.monthrange(year, month)[1]
+    )
+
+    return sum([
+        1 for x in range(1, date.day)
+        if datetime(year, month, x).weekday() < 5
+    ]) * eight_hour
```

Comment:
make no sense, multiple it by 28800 or move it to settings and call 'working_hours'